src/exposurelog/resolvers/delete_messages.py

```diff
@@ -17,8 +17,8 @@
 async def delete_messages(
     app: aiohttp.web.Application,
     _info: graphql.GraphQLResolveInfo,
-    **kwargs,
-) -> dict:
+    **kwargs: typing.Any,
+) -> typing.List[dict]:
```

Comment:
Now that we are on Python 3.8, I thought you could just use `list[dict]`

Reply:
Thanks. I'll make that change. Pity that `dict` doesn't work the same way (I tried).